.github/workflows/build-linux.yml

```diff
@@ -0,0 +1,29 @@
+
+name: Build on Ubuntu and JDK21
+
+on:
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-24.04
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK 21
```

Comment:
I'd use `Set up JDK` only. Version is already in `name`. Then in the future when using different `java-version`, it would be only needed to update in two places instead of three.

Reply:
I synced both workflows (win and ubuntu) based on your comments.